scripts/jenkins/ardana/ansible/vars/main.yml

```diff
@@ -8,3 +8,14 @@ rpm_verification_fixmes: "{{ verification_temp_dir }}/rpm-verify-modification-wh
 rpm_modifiable_config_files: "{{ verification_temp_dir }}/rpm-modifiable-config-files.txt"
 rpm_modifiable_config_file_regexps: "{{ verification_temp_dir }}/rpm-modifiable-config-file-regexps.txt"
 rpm_added_files_whitelist: "{{ verification_temp_dir }}/rpm-added-files-whitelist.txt"
+
+ca_file: "{{ trusted_ca_bundle }}"
+admin_project_name: "{{ KEY_API.vars.keystone_service.tenant }}"
```

Comment:
Variables such as `KEY_API.vars.keystone_service.tenant` are generated by the configuration processor and are only available for the Ardana ansible playbooks (i.e. those that are located in `~/openstack/ardana/ansible` on the deployer node). In other words, you will get an error because ansible cannot evaluate these variables.
If you need access to the glance and heat APIs in Ardana, I suggest you source the service.osrc file before running the `openstack` commands. This way, you won't need any of these variables.

Reply:
Done